### PR TITLE
docs: improve cloud docs

### PR DESCRIPTION
**Pull request by LucasJosefiak**

### CODE REVIEW

File: docs/assets/integrating-github/4_new.png
Comment (jenshor) [inline docs/assets/integrating-github/4_new.png:1]: This file should be renamed

File: docs/assets/screenshots/github_app_installation.png
Comment (jenshor) [inline docs/assets/screenshots/github_app_installation.png:1]: This file needs to removed from the PR

File: docs/widgetbook-cloud/projects.mdx
Comment (jenshor) [inline docs/widgetbook-cloud/projects.mdx:1]: This page has to be removed. Might also need to be removed from the docs.json file

File: docs/widgetbook-cloud/reviews.mdx [hunk 1]
```diff
@@ -4,22 +4,67 @@ Widgetbook Cloud introduces a _Review_ feature, that empowers Flutter teams to c
...
+## Concept of reviews
 
 A Widgetbook Review lets teams contrast UI changes between two versions. Moreover, it enables comparison of the current Widget version against its Figma design.
 - In a standard Pull Request, code differences between a `base` (previous version) and a `feature` (new version) branch are evaluated.
 - Similarly, in a Widgetbook Review, visual differences between the base build (last version) and feature build (current version) are assessed.
 For every pull request in your Git provider, our integration initiates a Widgetbook Cloud review. To facilitate this visual comparison, Widgetbook Cloud necessitates uploaded builds for both base and feature branches.
 
+## Publish a Review
+
+1. Make sure to incorporate the [widgetbook_generator package](https://pub.dev/packages/widgetbook_generator) version `{{ versions.generator }}`. No additional configuration needed.
```
Comment (jenshor) [inline docs/widgetbook-cloud/reviews.mdx:25]: Let's remove `No additional configuration needed.`

File: docs/widgetbook-cloud/reviews.mdx [hunk 2]
```diff
@@ -4,22 +4,67 @@ Widgetbook Cloud introduces a _Review_ feature, that empowers Flutter teams to c
...
+2. Integrate the [widgetbook package](https://pub.dev/packages/widgetbook) version `{{ versions.widgetbook }}`. No extra setup required.
```
Comment (jenshor) [inline docs/widgetbook-cloud/reviews.mdx:26]: Let's remove `No extra setup required.`

File: docs/widgetbook-cloud/reviews.mdx [hunk 3]
```diff
@@ -4,22 +4,67 @@ Widgetbook Cloud introduces a _Review_ feature, that empowers Flutter teams to c
...
+3. Employ the CLI from [widgetbook_cli package](https://pub.dev/packages/widgetbook_cli).
+4. Set up our GitHub app.
```
Comment (jenshor) [inline docs/widgetbook-cloud/reviews.mdx:28]: Let's put a link to the page here

File: docs/widgetbook-cloud/reviews.mdx [hunk 4]
```diff
@@ -4,22 +4,67 @@ Widgetbook Cloud introduces a _Review_ feature, that empowers Flutter teams to c
...
+5. Upload a build of your base branch, e.g. main. See [how to upload a build](https://docs.widgetbook.io/widgetbook-cloud/hosting).
```
Comment (jenshor) [inline docs/widgetbook-cloud/reviews.mdx:29]: main should be formatted as `main`. 

The link must be relative and must not contain `https:`

File: docs/widgetbook-cloud/reviews.mdx [hunk 5]
```diff
@@ -4,22 +4,67 @@ Widgetbook Cloud introduces a _Review_ feature, that empowers Flutter teams to c
...
+6. Add new or change existing Widgetbook use cases in your feature branch. Note: Without changed use cases, the review feature won't work, as there would be nothing to review.
```
Comment (jenshor) [inline docs/widgetbook-cloud/reviews.mdx:30]: Let's add another point here saying create a branch. 
Then we have what is currently 6 that says: Make a change, add a use case etc.

File: docs/widgetbook-cloud/reviews.mdx [hunk 6]
```diff
@@ -4,22 +4,67 @@ Widgetbook Cloud introduces a _Review_ feature, that empowers Flutter teams to c
...
+7. Optionally, [add addons and knobs](/widgetbook-cloud/reviews#how-to-set-up-addons-and-knobs) as well as [Figma design links](/widgetbook-cloud/reviews#how-to-add-a-figma-design-link).
+8. Upload a build for the feature branch with changed Widgetbook use cases.
```
Comment (jenshor) [inline docs/widgetbook-cloud/reviews.mdx:32]: Here the command is `widgetbook publish --api-key <API> --base-branch main`

File: docs/widgetbook-cloud/reviews.mdx [hunk 7]
```diff
@@ -4,22 +4,67 @@ Widgetbook Cloud introduces a _Review_ feature, that empowers Flutter teams to c
...
+9. Publish a review using this command: `widgetbook publish --api-key KEY --base-branch BASE-BRANCH`.
```
Comment (jenshor) [inline docs/widgetbook-cloud/reviews.mdx:33]: This should be part of what is currently 8.

**Review by jenshor — CHANGES_REQUESTED** [review]

**Review by jenshor — APPROVED** [review]

File: docs/assets/integrating-github/4.png
Comment (benitagrace) [inline docs/assets/integrating-github/4.png:1]: test

Comment (benitagrace) [reply docs/assets/integrating-github/4.png:1]: khkhkh

**Pull request merged**